### ISSUE TITLE
Accessiblity

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-services/contile#171]
Please provide more detail regarding this issue. 

(is this a task? Is it an ongoing concern? etc.)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/CONSVC-52)

**Comment by data-sync-user** [comment]
➤ David Durst commented:

Accessibility issues should be identified by the team that focuses on this from the client side.

**Comment by data-sync-user** [comment]
➤ David Durst commented:

Not actionable at this time.
